ts/screens/onboarding/PinScreen.tsx

```diff
@@ -133,19 +149,10 @@ class PinScreen extends React.PureComponent<Props, State> {
           ...this.state.pinState,
           state: "PinConfirmError"
         };
-        this.setState(
-          {
-            pinState: pinConfirmError,
-            errorDescription: I18n.t("onboarding.unlockCode.confirmInvalid")
-          },
-          () => {
-            // set focus on label to read about the error
-            setAccessibilityFocus(this.confirmationStatusRef);
-          }
-        );
-      } else if (this.state.pinState.state === "PinConfirmError") {
```

Comment:
by removing this case the reader reads "i due codici non coincidono" once
how to replicate:

1. insert 000000
2. insert 111111
3. it reads "i due codici non coincidono"
4. insert again 111111
5. it doesn't read "i due codici non coincidono"

Reply:
restored on 6f9e18c